<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CuttingDialog</class>
 <widget class="QDialog" name="CuttingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cutting</string>
  </property>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlaceObjectButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>560</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Place Object</string>
   </property>
  </widget>
  <widget class="QLabel" name="ObjectLeftLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>It remains to place objects:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="CountSpinBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="DimSheetLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>width</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>501</width>
     <height>511</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowResultButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>560</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show result</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BackButton</sender>
   <signal>clicked()</signal>
   <receiver>CuttingDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
